src/Platforms/OraclePlatform.php

```diff
@@ -522,11 +522,15 @@ private function addSuffix(string $identifier, string $suffix): string
      */
     private function getAutoincrementIdentifierName(Identifier $table): string
     {
-        $identifierName = $this->addSuffix($table->getName(), '_AI_PK');
+        $identifierName = $this->addSuffix($table->getName(), '_apk');
```

Comment:
As you only have 30 chars for identfiers in Oracle, I tried to keep the suffix as short as possible - especially if the hash value must be added too. With that in mind, the `_AI_PK` suffix is just a waste of characters and I tried to use only four characters like for `_seq`. Thus, it gets less likely that the hash value must be added too.

Reply:
This isn't strictly necessary to address the issue in question. It requires the same testing as in https://github.com/doctrine/dbal/pull/5000#pullrequestreview-811419639.

Reply:
Similar to #5000:

Migration works as expected, (unique) index and foreign key names are not affected because the modified addSuffix() method is only used to generate sequence and autoincrement trigger names.

Existing sequences for tables with names change their name because column name and maybe the the hash characters are added to the sequence name. Trigger names for autoincrement columns are also dropped and recreated with a new name.

All sequence names change and that's breaking because the sequence get dropped and recreated. Therefore, the next ID starts with 1 and inserting new data into a table which already contains data will fail.

Reply:
> Therefore, the next ID starts with 1 and inserting new data into a table which already contains data will fail.

In order for this to be accepted even in a major release, an upgrade path should be provided. I.e. what should the affected API consumer do handle this BC break on their side.